libpod/oci_conmon_linux.go

```diff
@@ -131,9 +131,9 @@ func newConmonOCIRuntime(name string, paths []string, conmonPath string, runtime
 		}
 		foundPath = true
 		runtime.path = path
-		logrus.Debugf("using runtime %q", path)
```

Comment:
Maybe keep this, but reword to "Found runtime %q"?

Reply:
This Issue was based on people complaining about too much info in debugging.  Should we drop this down to trace?

Reply:
SGTM